node/core/approval-voting/src/lib.rs

```diff
@@ -1534,6 +1553,15 @@ async fn handle_approved_ancestor<Context>(
 				unapproved.len(),
 				entry.candidates().len(),
 			);
+			if i == oldest_ancestor_idx && bits.len() > LOGGING_DEPTH_THRESHOLD {
```

Comment:
This doesn't print only up to the oldest `LOGGING_DEPTH_THRESHOLD` blocks.

Reply:
That would be all block starting with `ancestry.len() + 1 - LOGGING_DEPTH_THRESHOLD`